swim/memberlist.go

```diff
@@ -517,13 +555,28 @@ func (m *memberlist) Update(changes []Change) (applied []Change) {
 			} else {
 				// otherwise it can be applied to the memberlist
 
+				// prepare the change and collect if there is an outside
```

Comment:
I can't comment on line 554 but if we want to create memberChange-events for non-observable changes (see question above), we might want to append a `memberChange` about the local member on line 554 as well